src/ui/page/page.ios.ts

```diff
@@ -466,12 +466,18 @@ export default class PageIOS<TEvent extends string = PageEvents, TNative extends
         return self._largeTitleDisplayMode;
       },
       set largeTitleDisplayMode(value: HeaderBar['ios']['largeTitleDisplayMode']) {
-        if (__SF_UINavigationItem.instancesRespondToSelector('largeTitleDisplayMode')) {
+        //if (__SF_UINavigationItem.instancesRespondToSelector('largeTitleDisplayMode')) {
```

Comment:
We can remove commented code since there is no implementation of the "instancesRespondToSelector" method inside framework